### PR TITLE
Close issue #133

### PR DESCRIPTION
**Pull request by Solomon0519**
Add user guide for list functions and commands for switching parsers.

### CODE REVIEW

**Comment by NgZiXin** [comment]
Skeleton for MeetUp UG

File: src/main/java/seedu/address/logic/commands/meetup/FindCommand.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
  */
 public class FindCommand extends Command {
 
-    public static final String COMMAND_WORD = "findm";
+    public static final String COMMAND_WORD = "find";
```
Comment (ylchin) [inline src/main/java/seedu/address/logic/commands/meetup/FindCommand.java:18]: LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -74,6 +74,22 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
+### Switching parser modes : `switch`
+
+Switches the parser mode to the specified parser mode.
+
+Format: `switch PARSER_MODE`
+
+* Switches the parser mode to the specified `PARSER_MODE`.
```
Comment (ylchin) [inline docs/UserGuide.md:83]: very clear. good job!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -161,6 +177,77 @@ Clears all entries from the buyer list.
...
+Deletes the specified meet-up from the buyer list.
+
+Format: `delete INDEX`
+
+* Deletes the meet-up at the specified `INDEX`.
+* The index refers to the index number shown in the displayed meet-up list.
+* The index **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `list` followed by `delete 2` deletes the 2nd meet-up in the meet-up list.
```
Comment (ylchin) [inline docs/UserGuide.md:248]: we not using list anymore

**Pull request merged**